src/dev/i18n/extract_default_translations.test.js

```diff
@@ -34,40 +36,61 @@ const pluginsPaths = [
   path.join(fixturesPath, 'test_plugin_3'),
 ];
 
+jest.mock('../../../.localizationrc.json', () => ({
+  paths: {
+    plugin_1: 'src/dev/i18n/__fixtures__/extract_default_translations/test_plugin_1',
+    plugin_2: 'src/dev/i18n/__fixtures__/extract_default_translations/test_plugin_2',
+    plugin_3: 'src/dev/i18n/__fixtures__/extract_default_translations/test_plugin_3',
+  },
+  exclude: [],
+}));
+
 describe('dev/i18n/extract_default_translations', () => {
   test('extracts messages to en.json', async () => {
     const [pluginPath] = pluginsPaths;
-    await extractDefaultTranslations(pluginPath);
-
-    const extractedJSONBuffer = await readFileAsync(
-      path.join(pluginPath, 'translations', 'en.json')
-    );
+    await extractDefaultTranslations({ paths: [pluginPath], output: pluginPath });
+    const extractedJSONBuffer = await readFileAsync(path.join(pluginPath, 'en.json'));
 
-    await unlinkAsync(path.join(pluginPath, 'translations', 'en.json'));
-    await removeDirAsync(path.join(pluginPath, 'translations'));
+    await unlinkAsync(path.join(pluginPath, 'en.json'));
 
     expect(extractedJSONBuffer.toString()).toMatchSnapshot();
   });
 
   test('injects default formats into en.json', async () => {
     const [, pluginPath] = pluginsPaths;
-    await extractDefaultTranslations(pluginPath);
+    await extractDefaultTranslations({ paths: [pluginPath], output: pluginPath });
+    const extractedJSONBuffer = await readFileAsync(path.join(pluginPath, 'en.json'));
 
-    const extractedJSONBuffer = await readFileAsync(
-      path.join(pluginPath, 'translations', 'en.json')
-    );
-
-    await unlinkAsync(path.join(pluginPath, 'translations', 'en.json'));
-    await removeDirAsync(path.join(pluginPath, 'translations'));
+    await unlinkAsync(path.join(pluginPath, 'en.json'));
```

Comment:
__nit:__ By the way, I just realized that we may not need this magic with read/write/clean for real files, we can just mock `import fs from 'fs';` and use mock call arguments for snapshot testing and that's it, how does that sound?